zuul.d/templates.yaml

```diff
@@ -0,0 +1,12 @@
+- project-template:
```

Comment:
Project templates should go into https://github.com/ansible/zuul-jobs so we can keep them untrusted, this allows for pre-merge testing.